modules/installation-configuration-parameters.adoc

```diff
@@ -935,17 +935,25 @@ Additional {rh-openstack} configuration parameters are described in the followin
 |For compute machines, the size in gigabytes of the root volume. If you do not set this value, machines use ephemeral storage.
 |Integer, for example `30`.
 
+|`compute.platform.openstack.rootVolume.types`
+|For compute machines, the root volume's types.
+|A list of strings, for example, {`performance`, `standard`}. ^[1]^
+
 |`compute.platform.openstack.rootVolume.type`
-|For compute machines, the root volume's type.
-|String, for example `performance`.
+|For compute machines, the root volume's type. This property is deprecated.
+|String, for example, `performance`. ^[2]^
 
 |`controlPlane.platform.openstack.rootVolume.size`
 |For control plane machines, the size in gigabytes of the root volume. If you do not set this value, machines use ephemeral storage.
 |Integer, for example `30`.
 
+|`controlPlane.platform.openstack.rootVolume.types`
+|For control plane machines, the root volume's types.
+|A list of strings, for example, {`performance`, `standard`}. ^[1]^
```

Comment:
While not incorrect, I find the example misleading. Master machines must be equal in resources and performance; using "standard" and "performance" may imply differently. Also, masters are typically exactly three. Please consider:

```suggestion
|A list of strings, for example, {`performance-host1`, `performance-host2`, `performance-host3`}. ^[1]^
```